src/linear_algebra/free_module.lean

```diff
@@ -7,6 +7,9 @@ Authors: Riccardo Brasca
 import linear_algebra.direct_sum.finsupp
 import linear_algebra.std_basis
 import logic.small
+import ring_theory.finiteness
+import linear_algebra.matrix.to_lin
+import linear_algebra.std_basis
```

Comment:
Sorting the imports is not only my hobby, this also makes it clear that `linear_algebra.std_basis` is already imported :)
```suggestion
import linear_algebra.direct_sum.finsupp
import linear_algebra.matrix.to_lin
import linear_algebra.std_basis
import logic.small
import ring_theory.finiteness
```